### ISSUE TITLE
Fail to install an application from Microsoft Store because "missing bank account"

### ISSUE THREAD

**Issue by TechWatching** [microsoft/winget-cli#1043]
# Brief description of your issue

When I try to install an application from Microsoft Store it fails with a message indicating that my bank account is missing 😲.

# Steps to reproduce

- Enable installing ms store apps in winget settings.
- Launch powershell , and run the following command
```winget install -e -h --id Microsoft.WindowsTerminal -s msstore```

# Expected behavior

Winget find the app and install it.

# Actual behavior

winget is unable to verify/aquire package.
![image](https://user-images.githubusercontent.com/15186176/120175804-df7c7500-c206-11eb-8449-ecd24b1caf8b.png)

# Environment

```none
[winget --info]
Windows Package Manager v1.0.11451     
Windows: Windows.Desktop v10.0.19043.1023                                                                               
Package: Microsoft.DesktopAppInstaller v1.11.11451.0 

```

**Comment by denelon** [comment]
@TechWatching I'm guessing this may be a challenge with translation of the strings in the message.

**Comment by denelon** [comment]
@TechWatching are you logged into the Microsoft Store with your Microsoft Account (MSA) on this machine?

**Comment by TechWatching** [comment]
Yes I am logged in with my Microsoft Account on the Microsoft Store of this machine, even if the user logged in to my session is another account (azure ad / professional account). Could this be a problem ?

**Comment by denelon** [comment]
This looks like it is related to the account used between the Windows Package Manager and the *experimental* store implementation. We're still working through the edge cases here. I've marked this external as it currently isn't something that we can resolve in the Windows Package Manger client. As we get closer to being able to have stable/released store support this will be an issue that must be resolved.

**Comment by denelon** [comment]
@TechWatching

We've published the v1.1 [release candidate](https://github.com/microsoft/winget-cli/releases/tag/v1.1.12653) and updated our [troubleshooting guide](https://github.com/microsoft/winget-cli/tree/master/doc/troubleshooting).

Can you confirm if this is still happening?

**Comment by olivier-david** [comment]
In Windows 10 it was possible to connect to the store with a professional account but it's not possible anymore on windows 11.
If not connected it's not possible to defnie payment options, so I guess this is why we get this error.

**Comment by SuperITMan** [comment]
Ping! 
Please keep this open

**Comment by denelon** [comment]
@SuperITMan I removed the "Needs-Author-Feedback" label so the bot shouldn't try to close the issue.

**Comment by D0wn3r** [comment]
Hi !
Any update about that ? I'm trying to install Company portal with Winget but it failed because no bank account too

**Comment by denelon** [comment]
@D0wn3r

Are you logged into the Microsoft Store with an account?

If you are, do you get the same error message if you log out of the store?

**Comment by denelon** [comment]
Thanks for confirming. I've reached out to the team to see if we can isolate what causes this particular error.

**Comment by D0wn3r** [comment]
@denelon 
How to upgrade to this version please ?

**Comment by denelon** [comment]
You can just open the [MSIX bundle](https://github.com/microsoft/winget-cli/releases/download/v1.3.1741/Microsoft.DesktopAppInstaller_8wekyb3d8bbwe.msixbundle) from our releases page.

**Comment by D0wn3r** [comment]
Thank you very much @denelon, it works on my computer without being logged. I will try on my other computer that is AD joined registered to see.

**Comment by D0wn3r** [comment]
It works perfectly ! Nice job

**Comment by D0wn3r** [comment]
This issue can be closed I guess

**Comment by denelon** [comment]
I'm going to keep it open as I would like to understand what causes that specific error. We may run into it again.

**Comment by Rolodo** [comment]
Hi all, I'm having the same error while trying to install the company portal app as of today on a updated windows 11 computer.
I manage my Azure AD / Endpoint and we are trying to install it with powershell but we got that error.

here's the command I ran:
winget install --id "9WZDNCRFJ3PZ" --exact --source msstore --accept-package-agreements --accept-source-agreements

Thanks

**Comment by Rolodo** [comment]
I see that the issue will become stale should I open my own issue cause I could provide feedback if needed since I have the same issue.

Thanks

**Comment by ItzLevvie** [comment]
> I see that the issue will become stale should I open my own issue cause I could provide feedback if needed since I have the same issue.

You should because your issue is a bit different.

![image](https://user-images.githubusercontent.com/11600822/179590470-9bf7624e-9046-4d4e-9912-3ccd81738930.png)

**Comment by denelon** [comment]
We've released WinGet 1.4 today. I haven't seen additional reports of this kind since 1.3 was released. I'm not sure if it was a matter of translation between English to French for the "Bank Account" specific error or not, but the Windows Package Manager will check the user account if one is logged into the Microsoft Store, and if not, it will fall back to a device-based entitlement.
